File: iOS8_Advanced/2_2_ParseData/ParseData/ViewController.m
//
//  ViewController.m
//  1_3_ParseData
//
//  Created by Masayuki Nii on 2013/12/15.
//  Copyright (c) 2013年 msyk.net. All rights reserved.
//

#import "ViewController.h"
#import "ParseData-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    
    GenerateXML *xmlGenerator = [[GenerateXML alloc] init];
    [xmlGenerator testGenerateXML];

}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: iOS8_Advanced/SwiftAndObjC/SwiftAndObjC/ViewController.swift
//
//  ViewController.swift
//  SwiftAndObjC
//
//  Created by 新居雅行 on 2014/09/24.
//  Copyright (c) 2014年 msyk.net. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        let tc = TestClass();
        tc.name = "As I am";
        println(tc.deluxName("Great! "))
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

